src/components/MultipleAvatars.tsx

```diff
@@ -80,142 +72,156 @@ const avatarSizeToStylesMap = {
 
-    if (!props.icons.length) {
+    if (!icons.length) {
         return null;
     }
 
-    if (props.icons.length === 1 && !props.shouldStackHorizontally) {
+    if (icons.length === 1 && !shouldStackHorizontally) {
         return (
             <UserDetailsTooltip
-                accountID={props.icons[0].id}
-                icon={props.icons[0]}
+                accountID={icons[0].id}
+                icon={icons[0]}
                 fallbackUserDetails={{
-                    displayName: props.icons[0].name,
-                    avatar: props.icons[0].avatar,
+                    displayName: icons[0].name,
                 }}
             >
                 <View style={avatarContainerStyles}>
                     <Avatar
-                        source={props.icons[0].source}
-                        size={props.size}
+                        source={icons[0].source}
+                        size={size}
                         fill={themeColors.iconSuccessFill}
-                        name={props.icons[0].name}
-                        type={props.icons[0].type}
-                        fallbackIcon={props.icons[0].fallbackIcon}
+                        name={icons[0].name}
+                        type={icons[0].type}
+                        fallbackIcon={icons[0].fallbackIcon}
                     />
                 </View>
             </UserDetailsTooltip>
         );
     }
 
-    const oneAvatarSize = StyleUtils.getAvatarStyle(props.size);
-    const oneAvatarBorderWidth = StyleUtils.getAvatarBorderWidth(props.size).borderWidth;
+    const oneAvatarSize = StyleUtils.getAvatarStyle(size);
+    const oneAvatarBorderWidth = StyleUtils.getAvatarBorderWidth(size).borderWidth ?? 0;
     const overlapSize = oneAvatarSize.width / 3;
 
-    if (props.shouldStackHorizontally) {
+    if (shouldStackHorizontally) {
         // Height of one avatar + border space
         const height = oneAvatarSize.height + 2 * oneAvatarBorderWidth;
         avatarContainerStyles = StyleUtils.combineStyles([styles.alignItemsCenter, styles.flexRow, StyleUtils.getHeight(height)]);
     }
 
     return (
         <>
-            {props.shouldStackHorizontally ? (
-                _.map(avatarRows, (avatars, rowIndex) => (
+            {shouldStackHorizontally ? (
+                avatarRows.map((avatars, rowIndex) => (
                     <View
                         style={avatarContainerStyles}
+                        /* eslint-disable-next-line react/no-array-index-key */
                         key={`avatarRow-${rowIndex}`}
                     >
-                        {_.map([...avatars].splice(0, props.maxAvatarsInRow), (icon, index) => (
+                        {[...avatars].splice(0, maxAvatarsInRow).map((icon, index) => (
                             <UserDetailsTooltip
-                                key={`stackedAvatars-${index}`}
+                                key={`stackedAvatars-${icon.id}`}
```

Comment:
`icon.id` was undefined for Workspaces and hence created warnings. It was fixed here https://github.com/Expensify/App/pull/31484